stock_outgoing_shipment_report/report/stock_outgoing_shipment_report.py

```diff
@@ -43,7 +43,7 @@ class StockOutgoingShipmentReport(models.TransientModel):
     partner_phone = fields.Char("Phone")
     product_code = fields.Char("Product Code")
     product_name = fields.Char("Product Name")
-    case_qty = fields.Float("Case Quantity")
+    case_qty = fields.Char("Case Quantity")
```

Comment:
```suggestion
    case_qty = fields.Integer("Case Quantity")
```
I thought it would be more natural to use integer.

We just need to make this line in _validate_field_length():
```.py
                if rec[field] and len(rec[field]) > prop[1]:
```
like this:
```.py
                if rec[field] and len(str(rec[field])) > prop[1]:
```